src/reflect/scala/reflect/internal/SymbolTable.scala

```diff
@@ -51,6 +51,18 @@ abstract class SymbolTable extends macros.Universe
 
   val gen = new InternalTreeGen { val global: SymbolTable.this.type = SymbolTable.this }
 
+  trait ReflectStats extends BaseTypeSeqsStats
+                        with TypesStats
+                        with SymbolTableStats
+                        with TreesStats
+
+  /** Some statistics (normally disabled) set with -Ystatistics */
+  // This is a val so that we can override the `symbolTable` and `nodeCount` fields in `Global`
+  val statistics: ReflectStats = new {
```

Comment:
If this were an `object`, the definition of `symbolTable` and `nodeCount` would be `final` and the override in `Global` would fail.